alt_e2eshark/onnx_tests/models/onnx_zoo_models.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2024 Advanced Micro Devices, Inc.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+from e2e_testing.registry import register_test
+from ..helper_classes import (
+    OnnxModelZooDownloadableModel,
+    onnx_zoo_validated,
+    onnx_zoo_non_validated,
```

Comment:
I'd prefer to have these lists in this particular file, rather than in `helper_classes.py`.

I realize that you use these lists to determine some control flow in your model info class, but it may be better to have a class attribute bool `is_validated` that performs the same task. 

You could add an arg at the beginning of the `__init__` method, and then define in this file 

```python
validated_constructor =  lambda args, kwargs : OnnxModelZooDownloadableModel(True, *args, **kwargs)
non_validated_constructor =  lambda args, kwargs : OnnxModelZooDownloadableModel(False, *args, **kwargs)
```

Reply:
@zjgarvey this is actually rather difficult to do, as the constructor for the TestConfig only allows me to pass in the model_name and the model_constructor. What I have done for now is that moved these lists and the `model_path_map` to this file, and simply imported `model_path_map` in `helper_classes.py`.